test/test_distributions.py

```diff
@@ -866,6 +880,8 @@ def test_sample_gradient(jax_dist, sp_dist, params):
         "LKJ": ["concentration"],
         "LKJCholesky": ["concentration"],
         "StudentT": ["df"],
+        "AsymmetricLaplace": ["asymmetry"],
+        "AsymmetricLaplaceQuantile": ["quantile"],
```

Comment:
I think those parameters are reparametrizable. Did the test fail with those parameters? I can take a look if it is the case.

Reply:
The distributions did fail this test earlier, but it might have been for another reason. Will comment out these lines and rerun the test to confirm.

Reply:
When I comment out those lines, ALDQ fails with all three sets of parameters tested, but ALD still passes. I think it probably has to do with the calculation of `asymmetry` from reparameterized values of `scale` and/or `quantile`. The test shows two different error messages, one on the line for the calculation of `asymmetry`, giving a `ZeroDivisionError`, and the next two tests show an `AssertionError` where x has a nan value, and y has an imaginary complex number.

Reply:
> one on the line for the calculation of asymmetry, giving a ZeroDivisionError

Seems like this will happen at `asymmetry = (1 / ((1 / quantile) - 1)) ** 0.5` when quantile=0 or quantile=1. I will take a look.

Reply:
Seems like it's passing with lines commented out now, so will remove these...

Reply:
Sorry for that, it's a problem of the test (which assumes we have the right order of reparam parameters).